docs/src/css/custom.css

```
@@ -15,4 +15,87 @@
     --ifm-color-primary-lighter: #2da6d1;
     --ifm-color-primary-lightest: #00dcff;
     --ifm-color-secondary: #999999;
+    --ifm-alert-border-width: 6px;
+    --ifm-alert-background-color: #ffffff;
+    --ifm-panel-background-color: #ffffff;
+    --ifm-panel-box-shadow-color: #999999;
+    --ifm-code-color: var(--ifm-color-primary-darkest);
+  }
+
+  /******* COMMON *******/
+
+  html[data-theme='dark'] {
+    --ifm-panel-background-color: rgb(41, 45, 62);
+    --ifm-panel-box-shadow-color: var(--ifm-panel-background-color);
+    --ra-admonition-color: var(--ifm-color-black);
+    --ra-admonition-icon-color: var(--ifm-color-black);
+    --ifm-code-color: var(--ifm-color-primary-light);
+  }
+
+
+
+  /******* SIDE BAR STYLES *******/
+
+  .menu__link--sublist {
+    margin-top: 10px;
+    padding-bottom: 25px;
+    padding-top: 25px;
+  }
+
+
+
+  /******* MARKDOWN STYLES *******/
+
+  .alert {
+    border-radius: 0;
+    border: 0;
+    box-shadow: 0px 0px 10px 0px var(--ifm-panel-box-shadow-color);
+    border-width: 1px;
+    border-left: var(--ifm-alert-border-width) solid var(--ifm-alert-border-color);
+    background-color: var(--ifm-panel-background-color);
+    color: var(--ifm-font-color-secondary);
+  }
+
+  .admonition-icon svg {
+    fill: var(--ifm-font-color-secondary);
+  }
+
+  .alert a {
+    color: var(--ifm-font-color-secondary);
+  }
+
+  .markdown h4{
+    margin-top: 40px;
+    font-size: 20px;
+  }
+
+  .markdown h3{
+    margin-top: 45px;
+    font-size: 29px;
+  }
+
+  .markdown h2{
+    margin-top: 50px;
+    font-size: 35px;
+  }
+
+  .markdown .menuItems {
+    font-family: var(--ifm-font-family-monospace);
+    font-size: var(--ifm-code-font-size);
+    color: var(--ifm-color-primary-darkest);
+    display: inline-block;
+    background-color: #c8daf7;
```

```suggestion
    background-color: #8be9ff;
```
Let's adjust the coloring just a touch brighter and aqua, so that the text is more readable when users are not using dark mode.